troncos/traces/dd_exporter.py

```diff
@@ -1,6 +1,16 @@
 from typing import Sequence
 
+try:
+    from opentelemetry.exporter.otlp.proto.grpc import (  # type: ignore
+        trace_exporter as grpc_trace_exporter,
+    )
+except ImportError:  # pragma: no cover
+    grpc_trace_exporter = None
```

Comment:
This is how it's done in Aviso as well: https://github.com/kolonialno/aviso/blob/main/aviso-python/src/aviso/tracing/_tracer.py